map/light_framework.hpp

```diff
@@ -0,0 +1,85 @@
+#pragma once
+
+#include "map/user.hpp"
+
+#include "ugc/storage.hpp"
+
+#include "base/assert.hpp"
+
+namespace lightweight
+{
+struct LightFrameworkTest;
+
+enum RequestType
+{
+  Empty = 0u,
+  NumberOfUnsentUGC = 1u << 0,
+  UserAuthStatus = 1u << 1,
+  NumberOfUnsentEdits = 1u << 2,
+  All = NumberOfUnsentUGC | UserAuthStatus | NumberOfUnsentEdits
+};
+
+using RequestTypeMask = unsigned;
+
+class Framework
```

Comment:
Please add a comment about the intended usage of this class.

Reply:
Ok